cardano-cli/golden/files/golden/help/governance_action_create-no-confidence-motion.cli

```diff
@@ -0,0 +1,15 @@
+Usage: cardano-cli governance action create-no-confidence-motion --epoch EPOCH
+            --drep-cold-verification-key-file FILE
+            [--cost-model-file FILE]
```

Comment:
No cost-model or any other artifact is required.

Reply:
Removed cost model from all `governance action create` commands